msteams-platform/tabs/what-are-tabs.md

```diff
@@ -14,59 +14,45 @@ Tabs are client-aware webpages embedded in Microsoft Teams, Outlook, and Microso
 > [!IMPORTANT]
 > Custom tabs are available in Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD).
 
-The following image shows static tabs:
-
-# [Desktop](#tab/desktop)
-
-:::image type="content" source="../assets/images/tabs/personaltab.png" alt-text="Screenshot shows an example of a personal (static) tab in Teams desktop client.":::
-
-# [Mobile](#tab/mobile)
-
-:::image type="content" source="~/assets/images/tabs/personal-tab-mobile.png" alt-text="Example shows a mobile tab being added in Teams mobile client.":::
+There are two types of tabs available in Teams, personal and group. [Personal tabs](~/tabs/how-to/create-personal-tab.md) are also known as static tabs. Personal (static) tabs, along with personal-scoped bots, are part of the personal apps that are scoped to a single user and pinned to the left navigation bar for easy access. Personal (static) tabs in personal scope continue to work the same way. However, you can now [extend static tabs](~/tabs/how-to/create-personal-tab.md#extend-static-tabs-to-group-chat-channels-and-meetings) to group chat or meetings with a [customizable experience.](~/tabs/how-to/create-personal-tab.md#customizing-your-static-tab-in-chats-or-meetings)
```

Comment:
The two types of tabs are actually - "static" and "configurable". And they can be used in three different scopes - "personal", groupchat", and "teams".

Reply:
Updates